Library/Homebrew/brew.rb

```diff
@@ -15,7 +15,7 @@
 
 if ARGV == %w[--version] || ARGV == %w[-v]
   puts "Homebrew #{HOMEBREW_VERSION}"
-  puts "Homebrew/homebrew-core #{Homebrew.core_tap_version_string}"
+  puts "Homebrew/homebrew-core #{CoreTap.instance.version_string}"
```

Comment:
Looks like `CoreTap` hasn't been initialized yet at this point: https://travis-ci.org/Homebrew/brew/builds/164644934#L518-L520